### PR TITLE
[matio] Update to v1.5.19

### PR DESCRIPTION
**Pull request by traversaro**
- What does your PR fix? This PR updates matio to v1.5.19

- Which triplets are supported/not supported? Have you updated the CI baseline? The change should not affect the CI baseline.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)? Yes

### CODE REVIEW

File: ports/matio/portfile.cmake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO tbeu/matio
-    REF 64f793668e09216029447990702c145403d14375 # v1.5.17
+    REF ca56394e5672115df1981996ed12524d1d551259 # v1.5.19
     SHA512 bbb1c5f1f2609d329b69963f16ff8b60052922f83513d20f3ef687c04c4278536b88116c4727855de06f12ddf34103fe4be250012ae24706868490583ee90fde
```
Comment (JackBoosY) [inline ports/matio/portfile.cmake:5]:
```suggestion
    SHA512 67c239d8aabafaa935775f3b260ba0756c196b0d845ef5116365c9aa6a0b24dae70d92c4e74d5d43ae073ae0744f084f0f58ead864eb77c813d43dbbb3e4ec5e
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/matio/portfile.cmake (hunk 1 above)
Comment (traversaro) [reply ports/matio/portfile.cmake]: Sorry for the noise, I was still in the process of updating the PR.

**Review by JackBoosY — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**